examples/tools/autotools/autotoolstoolchain/string_formatter/run_example.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+echo "- AutotoolsToolchain: The toolchain generator for Autotools -"
```

Comment:
It would be fantastic if we could have this for Windows too:

- use a ``run_example.py`` script
- Condition the script to use .bat too in Windows
- Inject ``[tool_requires]`` in Windows for msys2, pkg_config

Could be a different example, or do we want the same example to be multi-platform?